package.json

```diff
@@ -62,24 +62,24 @@
     "gulp-uglify": "^1.2.0",
     "gulp-util": "^3.0.7",
     "jasmine": "^2.4.1",
-    "jasmine-core": "^2.2.0",
-    "karma": "^0.13.14",
-    "karma-chrome-launcher": "^0.2.1",
+    "jasmine-core": "^2.4.1",
+    "karma": "^0.13.20",
+    "karma-chrome-launcher": "^0.2.0",
     "karma-firefox-launcher": "^0.1.4",
     "karma-jasmine": "^0.3.6",
     "karma-mocha": "^1.2.0",
     "karma-safari-launcher": "^0.1.1",
-    "karma-sauce-launcher": "^0.2.10",
+    "karma-sauce-launcher": "^0.3.0",
     "karma-sourcemap-loader": "^0.3.6",
     "mocha": "^3.1.2",
     "nodejs-websocket": "^1.2.0",
     "promises-aplus-tests": "^2.1.2",
     "pump": "^1.0.1",
-    "systemjs": "^0.19.37",
+    "systemjs": "0.18.10",
     "ts-loader": "^0.6.0",
     "tslint": "^4.1.1",
     "tslint-eslint-rules": "^3.1.0",
-    "typescript": "2.0.10",
+    "typescript": "^2.1.5",
```

Comment:
can't upgrade typescript just yet. We need to do the upgrade in a separate PR and have separate version of Zones. Can you revert this one line?